src/server/worldserver/worldserver.conf.dist

```diff
@@ -1835,6 +1842,20 @@ Visibility.Notify.Period.InInstances  = 1000
 Visibility.Notify.Period.InBGArenas   = 1000
 
 #
+#    GuarderSight
+#        Description: Distance to see guard creatures
+#        Default:     50.0
+
+GuarderSight = 50.0
+
+#
+#    MonsterSight
+#        Description: Distance to see monster creatures
```

Comment:
Sure, I would like some help in deciding what to write in the description for the options.